### PR TITLE
Sequence diagram visualization

### LINKED ISSUE #536: Sequence diagram visualization fixes
- It is inconvenient to visualize only sequence diagrams or only other kind of diagrams separately.
- Inheritance from a sequence diagram is not supported yet: the diagram does not appear on the visualization dialog if it does not extend _Interaction_ or _SequenceDiagram_ class directly.

### PR DESCRIPTION
**Pull request by jtamas96**
Resolves #536, and related to #560

Contains:

- New dialog for sequence diagram visualization.

- Separated PreferencesManager from Papyrus export project.

- PlantUML export project no longer depends on Papyrus export project.

### CODE REVIEW

**Comment by kmozsi** [comment]
Jenkins, retest this, please.

File: dev/plugins/hu.elte.txtuml.utils.eclipse/src/hu/elte/txtuml/utils/eclipse/wizards/VisualizeTxtUMLPage.java [hunk 1]
```diff
@@ -75,78 +75,27 @@ public VisualizeTxtUMLPage() {
...
+				Logger.sys.error(e.getMessage());
+			}
+		} else {
+			createPage(parent, null);
+		}
 	}
 
 	/**
-	 * Returns the txtUML model layout classes
+	 * Returns the plantUML model layout classes
```
Comment (kmozsi) [inline dev/plugins/hu.elte.txtuml.utils.eclipse/src/hu/elte/txtuml/utils/eclipse/wizards/VisualizeTxtUMLPage.java]: I think this wizard is not only for plantUML visualization anymore.

**Comment by kmozsi** [comment]
The only work left to do is to create an icon for the new menu item.

**Comment by kmozsi** [comment]
Jenkins, retest this, please.

**Comment by nagyandras95** [comment]
retest this please

**Pull request merged**